test/TritonGPU/amd/amd-block-pingpong.mlir

```diff
@@ -888,3 +888,160 @@ module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 4 : i32, ttg.targ
     tt.return
   }
 }
+
+// -----
+//CHECK-LABEL: pingpong_medium_load_iter
+//CHECK: ttg.local_load
+//CHECK: ttg.local_load
+//CHECK: rocdl.sched.barrier
+//CHECK: tt.load
+//CHECK: rocdl.sched.barrier
+//CHECK: ttg.local_load
+//CHECK: ttg.local_load
+//CHECK: rocdl.sched.barrier
+//CHECK: tt.load
+//CHECK: rocdl.s.barrier
+//CHECK: rocdl.sched.barrier
+//CHECK: rocdl.s.setprio 1
+//CHECK: tt.dot
+//CHECK: rocdl.s.setprio 0
+//CHECK: gpu.barrier
+//CHECK: rocdl.sched.barrier
+//CHECK: ttg.local_store
+//CHECK: ttg.local_store
+//CHECK: gpu.barrier
+//CHECK: rocdl.sched.barrier
+//CHECK: rocdl.s.setprio 1
+//CHECK: tt.dot
+//CHECK: rocdl.s.setprio 0
+//CHECK: gpu.barrier
+//CHECK: rocdl.sched.barrier
+
+#blocked = #ttg.blocked<{sizePerThread = [1, 8], threadsPerWarp = [4, 8], warpsPerCTA = [8, 1], order = [1, 0]}>
+#blocked1 = #ttg.blocked<{sizePerThread = [1, 8], threadsPerWarp = [2, 16], warpsPerCTA = [8, 1], order = [1, 0]}>
+#mma = #ttg.amd_mfma<{versionMajor = 2, versionMinor = 0, warpsPerCTA = [4, 2], instrShape = [32, 32], isTransposed = true}>
+#shared = #ttg.swizzled_shared<{vec = 4, perPhase = 1, maxPhase = 16, order = [1, 0]}>
+#shared1 = #ttg.swizzled_shared<{vec = 1, perPhase = 1, maxPhase = 1, order = [1, 0]}>
+#smem = #ttg.shared_memory
+module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 8 : i32, ttg.target = "hip:gfx90a", "ttg.threads-per-warp" = 32 : i32} {
+  tt.func @pingpong_medium_load_iter(%arg0: !tt.ptr<f16> {tt.divisibility = 16 : i32}, %arg1: !tt.ptr<f16> {tt.divisibility = 16 : i32}, %arg2: !tt.ptr<f16> {tt.divisibility = 16 : i32}) {
```

Comment:
Can we simplify this test case? For example, do we need the epilogue in full? Looks to me we can delete L1028 - L1043 and make a function parameter to replace `%68`? Can you also see if you can trim down some other parts?